src/storage/src/storage_state/async_storage_worker.rs

```diff
@@ -62,42 +61,42 @@ pub enum AsyncStorageWorkerCommand<T: Timestamp + Lattice + Codec64> {
 /// Responses from [AsyncStorageWorker].
 #[derive(Debug)]
 pub enum AsyncStorageWorkerResponse<T: Timestamp + Lattice + Codec64> {
-    /// An `IngestionDescription` with a calculated, recent resume upper.
-    IngestDescriptionWithResumeUpper(
+    /// An `IngestionDescription` with recent as-of and resume upper frontiers.
+    FrontiersUpdated(
```

Comment:
I'm glad to do this as a fast-follow but these things being tuples to avoid writing docstrings is really annoying; should be a struct.